linq4j/src/main/java/org/apache/calcite/linq4j/tree/Evaluator.java

```diff
@@ -22,11 +22,11 @@
 /**
  * Holds context for evaluating expressions.
  */
-class Evaluator {
+public class Evaluator {
   final List<ParameterExpression> parameters = new ArrayList<>();
   final List<Object> values = new ArrayList<>();
 
-  Evaluator() {
+  public Evaluator() {
   }
```

Comment:
FYI: Seems to be the same as this PR:  https://github.com/apache/calcite/pull/1387

See the discusstion in this jira: https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-3260